### DISCUSSION TITLE
budi cli backup - Unable to access MinIO/S3

### DISCUSSION THREAD

**Discussion by bernhard-ma-adm** [Budibase/budibase#8662] [Help]
Hi all,
i'm hosting BudiBase on DigitalOcean with the droplet provided.

Now i want to backup it and migrate to another system.

I installed the cli manually (https://docs.budibase.com/docs/budibase-cli-setup).
DigitalOcean provides an .env-File
`
# Use the main port in the builder for your self hosting URL, e.g. localhost:10>
MAIN_PORT=10000

# This section contains all secrets pertaining to the system
# These should be updated
JWT_SECRET=_%some GUID%_
MINIO_ACCESS_KEY=_%some GUID%_
MINIO_SECRET_KEY=_%some GUIDs%_
COUCH_DB_PASSWORD=_%some GUID%_
COUCH_DB_USER=_%some GUID%_
REDIS_PASSWORD=_%some GUID%_
INTERNAL_API_KEY=_%some GUIDs%_

# This section contains variables that do not need to be altered under normal c>
APP_PORT=4002
WORKER_PORT=4003
MINIO_PORT=4004
COUCH_DB_PORT=4005
REDIS_PORT=6379
WATCHTOWER_PORT=6161
BUDIBASE_ENVIRONMENT=PRODUCTION

# An admin user can be automatically created initially if these are set
BB_ADMIN_USER_EMAIL=
`

The docker-compose.yaml is default from DigitalOcean. This is the part from the minIO-Service:

`minio-service:`
`    restart: unless-stopped`
`    image: minio/minio`
`    volumes:`
`      - minio_data:/data`
`    environment:`
`      MINIO_ACCESS_KEY: ${MINIO_ACCESS_KEY}`
`      MINIO_SECRET_KEY: ${MINIO_SECRET_KEY}`
`      MINIO_BROWSER: "off"`
`    command: server /data --console-address ":9001"`
`    healthcheck:`
`      test: ["CMD", "curl", "-f", "http://localhost:9000/minio/health/live"]`
`      interval: 30s`
`      timeout: 20s`
`      retries: 3`

Now i run the Backup:
`
budi backups --export ~/backup/20221112-forMig.tar.gz --env ~/.env
`
and i get this output:
`CouchDB Export`
` ████████████████████████████████████████ 100% | ETA: 0s | 10/10`
`S3 Export`
`Error - Error: Unable to access MinIO/S3 - check environment config.`
What am i doing wrong?

**Comment by bernhard-ma-adm** [comment]
Hi,
i could solve my problem by myself :)
After try to fetch the minIO-Service per curl inside the Docker-Container i changed MINIO_ACCESS_KEY to MINIO_ROOT_USER and MINO_SECRET_KEY to MINIO_ROOT_PASSWORD and make a backup without the .env-File and paste the values manually.

But now...after the import of my Backup on another system, i cannot access the apps.
After the import Budibase was automatically updated to the actual version and when i log in with my Admin-Account i see all my apps. But when i want to manage/edit/access the app:
`Error initalising app:`
The called resource (`https://bb.blitsc.de/api/app_dev_9c4b431d5e2b487593d9e5fd411dbf29/components/definitions`) is not found: HTTP 404.
so i cannot delete the app and nothing else.
I tried to export my App on the old system and import it in the new system - but...`Error creating app`

A new app is stuck in the same way. Restart of the Container and System didn't solve the problem...
What can i do to get my Apps work?
Thx!!!

**Reply by melohagan** [reply]
Hey @bernhard-mischokacademy 

You may need to wipe your docker volumes and try importing your apps again. This will however clear all Budibase data including users, etc.
